conductor/__init__.py

```diff
@@ -5,23 +5,14 @@
 import tempfile
 import warnings
 
-from conductor.lib import common, loggeria, version_check, wizard
+from conductor.lib import common, config, loggeria, version_check, wizard
 
 #The version string is updated by the build system.
 #Do not modify the following line.
 #__version__="0.0.0"
 
 # Read the config yaml file upon module import
-try:
-    CONFIG = common.Config().config
-except ValueError:
-    # wizard.run()
-    CONFIG = common.Config().config
-
-# IF there is log level specified in config (which by default there should be), then set it for conductor's logger
-log_level = CONFIG.get("log_level")
-if log_level:
-    loggeria.set_conductor_log_level(log_level)
+CONFIG = config.load_config()
```

Comment:
Just a thought here:   I'm wondering if we may still want to try/catch the loading of the config.  Just so that we have the opportunity to give the customer a more human-friendly hint as to what the problem is, e.g.
```
try:
    CONFIG = config.load_config()
except:
    print "ERROR: Failed to load Conductor config file"
    raise

```
*Note that a logger handler hasn't been set up at this point, hence the use of `print`

I don't feel strongly about this, so whatever you guys think